src/main/resources/workflows/0.1.0/messages_en.properties

```diff
@@ -0,0 +1,36 @@
+#Pipeline Info Properties
+#Wed Dec 04 16:34:39 CST 2019
+pipeline.parameters.modal-title.star_amr=STAR_AMR Pipeline Parameters
+workflow.STAR_AMR.description=A pipeline to search microbial genomes for antimicrobial resistance genes (AMR). Uses the tool StarAMR.
+pipeline.h1.STAR_AMR=STAR_AMR Pipeline
+pipeline.title.STAR_AMR=Pipelines - STAR_AMR
+workflow.STAR_AMR.title=STAR_AMR Pipeline
+#Tool Parameters - Tool: shovill - Workflow Step #: 1
+#Wed Dec 04 16:34:39 CST 2019
+pipeline.parameters.amr-detection.shovill-1-adv.nocorr=Disable post-assembly correction
```

Comment:
Could you change the string `pipeline.parameters.amr-detection` here to `pipeline.parameters.star_amr` to match the set of parameters below. Otherwise IRIDA won't be able to find the text (since all it's looking for something with `star_amr` in the key instead of `amr-detection`).